src/app/ReadClient.h

```diff
@@ -178,19 +179,19 @@ class ReadClient : public Messaging::ExchangeDelegate
     NodeId GetPeerNodeId() const { return mPeerNodeId; }
     bool IsReadType() { return mInteractionType == InteractionType::Read; }
     bool IsSubscriptionType() const { return mInteractionType == InteractionType::Subscribe; };
-    CHIP_ERROR SendStatusResponse(CHIP_ERROR aError);
+    CHIP_ERROR SendStatusResponseMessage(CHIP_ERROR aError);
 
 private:
     friend class TestReadInteraction;
     friend class InteractionModelEngine;
 
     enum class ClientState
     {
-        Uninitialized = 0,         ///< The client has not been initialized
-        Initialized,               ///< The client has been initialized and is ready for a SendReadRequest
-        AwaitingInitialReport,     ///< The client is waiting for initial report
-        AwaitingSubscribeResponse, ///< The client is waiting for subscribe response
-        SubscriptionActive,        ///< The client is maintaining subscription
+        Uninitialized = 0,                ///< The client has not been initialized
+        Initialized,                      ///< The client has been initialized and is ready for a SendReadRequestMessage
+        AwaitingInitialReport,            ///< The client is waiting for initial report
+        AwaitingSubscribeResponseMessage, ///< The client is waiting for subscribe response
```

Comment:
I'm not sure this use should be renamed....

Reply:
sure, rename it back.